### PR TITLE
Pt 158452869 channel ws events deposit withdraw

### PR DESCRIPTION
**Pull request by velzevur**
[deposit/withdraw WS events](https://www.pivotaltracker.com/story/show/158452869)

Still TODO
- [x] Add a line in the release notes
- [x] Add description in the protocol

### CODE REVIEW

**Comment by lucafavatella** [comment]
Potential conflict with NG re integration test suite so labelling as WIP for now.

**Comment by velzevur** [comment]
Rebased on top of NG changes.
Must move the release note line to the release18 when it is planted.

File: docs/release-notes/RELEASE-NOTES-0.17.0.md [hunk 1]
```diff
@@ -4,6 +4,7 @@
 It:
 * Stores the chain state in the database in a more compact way. This affects the persisted chain.
 * Changes the way we build chain, by electing a leader and introducing Key and Micro blocks. It is followed by new candidates generation modules, new serialized formats and enhanced sync layer.
+* Enriches channels' websocket API with deposit and withdrawal sequences.
```
Comment (lucafavatella) [inline docs/release-notes/RELEASE-NOTES-0.17.0.md]: Ok here until decided not in scope for 17.

**Comment by velzevur** [comment]
Aiming at merging in 17; probably will have to rebase once #1279 is merged (will conflict in release notes)

**Comment by velzevur** [comment]
Rebased on top of master and resolved conflicts

**Comment by velzevur** [comment]
Rebased on top of master and resolved conflicts

**Pull request merged**